pcapng_exporter/mapping.cpp

```diff
@@ -47,13 +50,15 @@ namespace pcapng_exporter {
 		target.inf_name = packet_interface.name
 			? std::optional(std::string(packet_interface.name))
 			: std::nullopt;
+		target.direction = std::nullopt;
```

Comment:
TECMP provides Tx / Rx, pass it through

Reply:
passed it on as a value in the header flag and then affected it to the target